sdk/healthinsights/health-insights-radiologyinsights-rest/review/health-insights-radiologyinsights.api.md

```diff
@@ -0,0 +1,1090 @@
+}
+
+// @public
+export interface CompleteOrderDiscrepancyInferenceOutput extends RadiologyInsightsInferenceOutputParent {
+    kind: "completeOrderDiscrepancy";
+    missingBodyPartMeasurements?: Array<CodeableConceptOutput>;
+    missingBodyParts?: Array<CodeableConceptOutput>;
+    orderType: CodeableConceptOutput;
+}
+
+// @public
+export interface Condition extends DomainResourceParent {
+    abatementAge?: Quantity;
+    abatementDateTime?: string;
+    abatementPeriod?: Period;
+    abatementRange?: Range;
+    abatementString?: string;
+    bodySite?: Array<CodeableConcept>;
+    category?: Array<CodeableConcept>;
+    clinicalStatus?: CodeableConcept;
+    code?: CodeableConcept;
+    encounter?: Reference;
+    identifier?: Array<Identifier>;
+    note?: Array<Annotation>;
+    onsetAge?: Quantity;
+    onsetDateTime?: string;
+    onsetPeriod?: Period;
+    onsetRange?: Range;
+    onsetString?: string;
+    recordedDate?: string;
+    resourceType: "Condition";
+    severity?: CodeableConcept;
+    stage?: Array<ConditionStage>;
+    verificationStatus?: CodeableConcept;
+}
+
+// @public
+export interface ConditionOutput extends DomainResourceOutputParent {
+    abatementAge?: QuantityOutput;
+    abatementDateTime?: string;
+    abatementPeriod?: PeriodOutput;
+    abatementRange?: RangeOutput;
+    abatementString?: string;
+    bodySite?: Array<CodeableConceptOutput>;
+    category?: Array<CodeableConceptOutput>;
+    clinicalStatus?: CodeableConceptOutput;
+    code?: CodeableConceptOutput;
+    encounter?: ReferenceOutput;
+    identifier?: Array<IdentifierOutput>;
+    note?: Array<AnnotationOutput>;
+    onsetAge?: QuantityOutput;
+    onsetDateTime?: string;
+    onsetPeriod?: PeriodOutput;
+    onsetRange?: RangeOutput;
+    onsetString?: string;
+    recordedDate?: string;
+    resourceType: "Condition";
+    severity?: CodeableConceptOutput;
+    stage?: Array<ConditionStageOutput>;
+    verificationStatus?: CodeableConceptOutput;
+}
+
+// @public
+export interface ConditionStage {
+    summary?: CodeableConcept;
+    type?: CodeableConcept;
+}
+
+// @public
+export interface ConditionStageOutput {
+    summary?: CodeableConceptOutput;
+    type?: CodeableConceptOutput;
+}
+
+// @public
+export interface ContactDetail extends Element {
+    name?: string;
+    telecom?: Array<ContactPoint>;
+}
+
+// @public
+export interface ContactDetailOutput extends ElementOutput {
+    name?: string;
+    telecom?: Array<ContactPointOutput>;
+}
+
+// @public
+export interface ContactPoint {
+    period?: Period;
+    rank?: number;
+    system?: string;
+    use?: string;
+    value?: string;
+}
+
+// @public
+export interface ContactPointOutput {
+    period?: PeriodOutput;
+    rank?: number;
+    system?: string;
+    use?: string;
+    value?: string;
+}
+
+// @public
+function createClient(endpoint: string, credentials: KeyCredential, options?: ClientOptions): AzureHealthInsightsClient;
+export default createClient;
```

Comment:
@joheredi do we use `export default` in generated clients? How come? And Can we move away from it or is it too late? https://github.com/isaacs/tshy?tab=readme-ov-file#handling-default-exports

Reply:
I also want to know about the `export default`

Reply:
@maorleger, indeed, we generate default exports in our RLCs, focusing on the client factory function as the core component of the library. This decision was made to streamline the user experience, emphasizing the primary functionality while still offering named exports for other helper functions.

I appreciate you highlighting this topic. It's certainly worth discussing internally. Since we've already released several RLCs to GA, transitioning away from this would demand strong reasons, but I'm open to discussing potential changes and their implications.